modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -648,8 +648,8 @@ export class Wallet implements IWallet {
    * @param {Number} params.maxFeeRate - upper limit for feeRate in satoshis/kB
    * @param {Number} params.maxFeePercentage - the maximum relative portion that you're willing to spend towards fees
    * @param {Number} params.feeTxConfirmTarget - estimate the fees to aim for first confirmation with this number of blocks
-   * @param {Number} params.minValue - the minimum value of unspents to use in satoshis
-   * @param {Number} params.maxValue - the maximum value of unspents to use in satoshis
+   * @param {Number | String} params.minValue - the minimum value of unspents to use in satoshis
```

Comment:
```suggestion
   * @param {number | string} params.minValue - the minimum value of unspents to use in satoshis
```

Reply:
the param should match the type, can you also make this change to all places you modified also

Reply:
added at other places too